src/libraries/System.Text.Json/tests/Serialization/ReferenceHandlingTests.Deserialize.cs

```diff
@@ -1144,7 +1148,7 @@ public static void ReferenceObjectsShouldNotContainMoreProperties()
             }";
 
             ex = Assert.Throws<JsonException>(() => JsonSerializer.Deserialize<Employee>(json, s_deserializerOptionsPreserve));
-            Assert.Equal("$.Manager", ex.Path);
+            Assert.Equal("$.Manager.$ref", ex.Path);
```

Comment:
Same here, I think it should be `$.Manager.$id`.

Reply:
Per comment above, `$Manager` seems to be the most correct + easiest to implement.